lib/auth.dart

```diff
@@ -285,4 +285,13 @@ Future<AccessCredentials> refreshCredentials(
       idToken: idToken);
 }
 
+/// Available response types that can be requested when using the implicit
+/// browser login flow.
+enum ResponseType {
+  code,
```

Comment:
If you want to be really nice, add a short doc-comment to each of these entrie s :)

Just a one-liner would go far...